azure/azure-how-to/instances/find-ubuntu-images.rst

```diff
@@ -80,6 +100,24 @@ Ubuntu Pro Offers
 
 To learn more about Ubuntu Pro on Azure: `Ubuntu Pro for Azure <https://ubuntu.com/azure/pro>`_
 
+Ubuntu 24.04 LTS - Noble Numbat
+++++++++++++++++++++++++++++++++++++++
+
+| Quick start: `Ubuntu Pro 24.04 LTS on Azure <https://portal.azure.com/#create/canonical.ubuntu-24_04-ltsubuntu-pro>`_
+
+
```

Comment:
Will the second empty line adversely impact the presentation of the rendered page?

Reply:
@danpdraper The empty line will not impact the presentation. If you want to take a look at the rendered version of the PR, you click on 'Show all checks' below, and then on 'details' for the azure check:
![image](https://github.com/canonical/ubuntu-cloud-docs/assets/123362247/db62cb8b-eb4d-4a51-aff5-e7943bec17ac)

In this case the rendered version is at https://canonical-ubuntu-documentation-library--200.com.readthedocs.build/azure/en/200/azure-how-to/instances/find-ubuntu-images/